yellowbrick/regressor/residuals.py

```diff
@@ -354,6 +370,10 @@ class ResidualsPlot(RegressionScoreVisualizer):
     line_color : color, default: dark grey
         Defines the color of the zero error line, can be any matplotlib color.
 
+    alpha : float, default: 0.75
```

Comment:
I think this is a good compromise for a default alpha between 0.5 and 0.9 (the original train and test alphas, respectively). Good choice!